test/functional/app/api/test_transfer_usage_api.py

```diff
@@ -30,3 +31,29 @@ def test_transfer_usage_api(test_client, authed_sempo_admin_user,name,icon,trans
                                 content_type='application/json', follow_redirects=True)
 
     assert response.status_code == status_code
+
+    # Ensure that the posting a duplicate transfer ussage results in a 400
+    response = test_client.post('/api/transfer_usage/',
+                                headers=dict(Authorization=auth_token + '|' + tfa_token, Accept='application/json'),
```

Comment:
i may have commented on this elsewhere, or maybe it was someone else's PR, but there's a utility function for making this easier: `get_complete_auth_token` in utils > auth.py